src/components/webview/WebviewResolver.ts

```diff
@@ -49,6 +50,27 @@ export class WebviewResolver {
 			`${webviewName}.js`,
 		]);
 
+		// TODO: enable importing chunks on demand
+		// find files that end with .chunk.js
+		const chunkFiles = fs
```

Comment:
@henoktsegayetoptal @DmytroHryshyn did you discuss the chunkification?

Reply:
I haven't actually , here I am loading chunks at start

Reply:
we are generating chunks from monaco loader